### PR TITLE
added scripts for running ceres system and e2et.

### PR DESCRIPTION
**Pull request by shabd67**
# Resolves

[CERES-698](https://rackspace.atlassian.net/browse/CERES-698)

# What

added scripts for running ceres system and e2et.

### CODE REVIEW

**Review by rohitsngh27 — DISMISSED** [review]
LGTM

File: test-scripts/e2et-script.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import requests, sys
+
+response = requests.get(sys.argv[1])
+result = response.text
+print(result)
+if result != "SUCCESS" :
+      raise Exception("Ceres End-to-end Test Case Failed..!!")
```
Comment (iWebi) [inline test-scripts/e2et-script.py:7]:
```suggestion
      raise Exception("Ceres end-to-end tests failed..!!")
```

File: test-scripts/system-test-script.py [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import requests, sys
+
+response = requests.get(sys.argv[1])
+result = response.text
+print(result)
+if result != "SUCCESS" :
+      raise Exception("Ceres System TestCase Failed..!!")
```
Comment (iWebi) [inline test-scripts/system-test-script.py:7]:
```suggestion
      raise Exception("Ceres System tests failed..!!")
```

**Review by iWebi — DISMISSED** [review]
Am not sure where these scripts are used in pipeline. These scripts look identical to me except the message is different.

**Comment by shabd67** [comment]
> Am not sure where these scripts are used in pipeline. These scripts look identical to me except the message is different.

had a word with Gaurav and I have explained him whole scenario. he is looking into how he can use these scripts. will create a separate story for Gaurav.

**Review by iWebi — APPROVED** [review]

**Pull request merged**